### PR TITLE
kingarthur's updates for Factorio 0.15

### PR DESCRIPTION
**Pull request by dgw**
Tweaks to research & entity data from @kingarthur91 (tagged as the commit author). I'm just the committer, filtering out the line-ending changes and such that can be really hard to deal with if you have little Git experience. (Easy solution: Put `* text=auto` in a `.gitattributes` file at the repo root, and Git will sort it out for you.) Source files for the updates I committed here came from [this forum post](https://forums.factorio.com/viewtopic.php?p=296120#p296120) by kingarthur.

I took the liberty of holding back changes that appeared related to renaming the mod "Warehousing v15", but I can commit and add that set of changes in just a few keystrokes if @Anoyomouse wants them. I operated under the (possibly incorrect) assumption that these changes will go into an updated version of the mod currently published on the mod portal.

Attempted to do the sensible thing with version numbers, and matched the values in `info.json` and `control.lua`. I apologize if that is incorrect.

### CODE REVIEW

**Comment by Anoyomouse** [comment]
Hey

Thanks for doing this for me ...

I don't think needing high tech packs are required for the warehouses, they don't ahve anything fancy to them, jsut lots of storage space, so i'd personally prefer cutting that requirement

otherwise i'll pull it into the pack

File: prototypes/technology.lua [hunk 1]
```diff
@@ -70,14 +70,16 @@ data:extend(
...
 		unit =
 		{
 			count = 25,
 			ingredients = {
 				{ "science-pack-1", 2},
 				{ "science-pack-2", 2},
-				{ "science-pack-3", 1}
+				{ "science-pack-3", 1},
+				{ "production-science-pack", 1},
+				{ "high-tech-science-pack", 1}
```
Comment (Anoyomouse) [inline prototypes/technology.lua]: just concerned here, there's nothing high tech in the warehouses

Comment (dgw) [inline prototypes/technology.lua]: Done and updated.

**Comment by dgw** [comment]
Did I miss something that needs to be fixed?

**Comment by Anoyomouse** [comment]
Hey, sorry about that, i've been buzy with work, and other things and have not had time to merge your changes.

I've done that now, however i still need to bundle it for a release on the mod portal

Any chance you can send me a message to pull in these changes so i can quickly deploy it to the mod portal?

**Comment by dgw** [comment]
No worries, was just wondering if I'd fixed up the research wrong or something.

What do you need in the message? (I presume on the Factorio forums.)

**Pull request merged**